### PR TITLE
test: Code Coverage 100% team advance card component

### PR DESCRIPTION
**Pull request by CODEX108**
ClickUp:  https://app.clickup.com/t/85zru6cwy
image

### CODE REVIEW

File: src/app/fyle/team-advance/team-adv-card/team-adv-card.component.spec.ts [hunk 1]
```diff
@@ -1,26 +1,65 @@
...
+      );
+      expect(advanceRequestServiceSpy.getInternalStateAndDisplayName).toHaveBeenCalledTimes(2);
+      expect(teamAdvCardComponent.internalState).toEqual({ state: 'inquiry', name: 'Sent Back' });
+    });
+  });
+
+  it('should emit an event when onGoToAdvances() is called', () => {
+    spyOn(teamAdvCardComponent.gotoAdvance, 'emit');
+    teamAdvCardComponent.onGoToAdvances();
+    expect(teamAdvCardComponent.gotoAdvance.emit).toHaveBeenCalledWith(teamAdvCardComponent.advanceRequest);
```
Comment (Dimple16) [inline src/app/fyle/team-advance/team-adv-card/team-adv-card.component.spec.ts:63]: `toHaveBeenCalledOnceWith`

File: src/app/fyle/team-advance/team-adv-card/team-adv-card.component.spec.ts [hunk 2]
```diff
@@ -1,26 +1,65 @@
...
+  });
+
+  describe('OnInit', () => {
+    it('should set showDate to false when advanceRequest was created on the same day as prevDate', () => {
+      teamAdvCardComponent.prevDate = new Date('2023-05-16T06:22:47.058Z');
+      fixture.detectChanges();
+      teamAdvCardComponent.ngOnInit();
+      expect(teamAdvCardComponent.currencySymbol).toEqual('$'); // Assuming USD is the currency code
+      expect(teamAdvCardComponent.showDate).toBeFalse();
+      expect(advanceRequestServiceSpy.getInternalStateAndDisplayName).toHaveBeenCalledWith(
```
Comment (Dimple16) [inline src/app/fyle/team-advance/team-adv-card/team-adv-card.component.spec.ts:52]: `toHaveBeenCalledOnceWith`

**Review by Dimple16 — APPROVED** [review]
minor comments, fix and merge

File: src/app/fyle/team-advance/team-adv-card/team-adv-card.component.spec.ts [hunk 3]
```diff
@@ -1,26 +1,65 @@
...
+        teamAdvCardComponent.advanceRequest
+      );
+      expect(advanceRequestServiceSpy.getInternalStateAndDisplayName).toHaveBeenCalledTimes(2);
```
Comment (CODEX108) [inline src/app/fyle/team-advance/team-adv-card/team-adv-card.component.spec.ts:55]: I have added the assertion here

**Pull request merged**